ProductApp/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+spring.application.name=ProductApp
+server.port=8000
+
+spring.datasource.url=jdbc:mysql://localhost:3306/ManagementProduct
```

Comment:
We cannot test this app since we don't have the DB and the credentials, please upload the docker files for having this container

Reply:
done